kong/plugins/aws-lambda/schema.lua

```diff
@@ -14,5 +14,6 @@ return {
     log_type = {type = "string", required = true, default = "Tail", 
                        enum = {"Tail", "None"}},
     port = { type = "number", default = 443 },
+    unhandled_response = {type = "number", default = -1, required = true},
```

Comment:
> What happens to numeric values when unset?

They would just be `nil`. A validation function that ensure a valid HTTP status `> 0 && <= 999` would also be nice. As we know, HTTP clients don't have a good time when the status is not included in that range.

I like the current approach of only setting `ngx.status` when `unhandled_status` is not `nil`.